exercises/complex-numbers/complex-numbers.spec.js

```diff
@@ -0,0 +1,236 @@
+import ComplexNumber from './complex-numbers.js';
+
+xdescribe('Imaginary unit', () => {
```

Comment:
We probably should make the `describe` nesting a little less complicated here.  Here's what I'm thinking:

1. I think it's nice how test blocks are grouped conceptually. But, we don't have any other examples in the entire track with nested `describe` blocks.  Nothing wrong with showing people the possibilities, but that leads me to thought (2):

2. Sometimes we're using `xdescribe` to skip an entire block, but once the block is un-skipped, we have multiple unskipped tests, and possibly multiple failures for the user.  For this reason, I think it would be better to use `xtest`, even inside the `xdescribe` blocks.  Let them proceed by passing one test at a time, instead of an entire block.

Reply:
Agree. All `test` blocks but the first one should be skipped with `xtest`, even if they are nested in a `describe` block.

About nesting `describe` blocks, the philosophy of the track is to keep it as simple as possible. By default, there shouldn't be any nested `describe`. Exercism users just have to make individual `test`s blocks one by one. I think nesting `describe`s adds noise to that.

Anyway, we can discuss if there is a reason to nest `describe`s.

Reply:
I grouped the tests in `decribe` blocks as they where grouped in the `canonical-data.json` file in the problem specifications. I however agree that in order to keep things simple for the users, we should only use one `describe` block and just skip tests by using `xtest`.

Reply:
@MattH-be @rchavarria I don't have a problem with the nested `describe`s, but I think the suite looks better now, easier to understand.  👍